### PR TITLE
LibWeb: Fetch (continued)

### PR DESCRIPTION
**Pull request by linusg**

### CODE REVIEW

**Review by kennethmyhra — APPROVED** [review]
This looks in general good to me! 💯

File: Userland/Libraries/LibWeb/Fetch/BodyInit.cpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022, Linus Groh <linusg@serenityos.org>
+ * Copyright (c) 2022, Kenneth Myhra <kennethmyhra@serenityos.org>
```
Comment (davidot) [inline Userland/Libraries/LibWeb/Fetch/BodyInit.cpp:3]: I assume this is intentional? Just asking since there is no co-author on the commit

**Review by davidot — APPROVED** [review]
Just one copyright question, code looks good to me

File: Userland/Libraries/LibWeb/Fetch/BodyInit.cpp (hunk 1 above)
Comment (linusg) [reply Userland/Libraries/LibWeb/Fetch/BodyInit.cpp:3]: `extract_body()` was written by Kenneth originally :)

**Pull request merged**